react/features/base/conference/reducer.js

```diff
@@ -209,7 +209,10 @@ function _lockStateChanged(state, action) {
         return state;
     }
 
-    return setStateProperty(state, 'locked', action.locked || undefined);
+    return setStateProperties(state, {
+        locked: action.locked || undefined,
+        roomLockPassword: action.locked ? state.roomLockPassword : null
```

Comment:
It bothers me a little bit that one is called "locked" and doesn't have "room" in the name and the other "roomLockPassword". Let's either rename locked to "roomLocked" or rename "roomLockPassword" to "lockPassword".